test/ng/animateCssSpec.js

```diff
@@ -31,6 +31,28 @@ describe("$animateCss", function() {
       expect(copiedOptions).toEqual(initialOptions);
     }));
 
+    it("should not create a copy of the provided options if they have already been prepared earlier",
```

Comment:
Is it possible to provoke the unwanted render behavior instead? That the options are copied seems like an implementation detail to me.